CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 5.0.0
```

Comment:
Why the major version update?

Reply:
new endpoint was added: Create-empty-index

Reply:
Oh ok. I think in the past we have bumped the major version to indicate a breaking change. Adding a new endpoint doesn't really break backwards compatibility though.

Reply:
@lcarva It was my bad. While filing the issue for the release, I assumed the change for `related_bundle_images`  (#270) would be included and so I filed for 5.0.0 because that could potentially break some code bases that expect a specific response and not extra params (like the problems we faced with Pub CI breaking when we add a new param to the response)

@pbortlov I believe a 4.2.0 should suffice?

Reply:
thanks